addon/-private/properties/alias.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+
+import {
+  getProperty,
+  objectHasProperty
+} from '../helpers';
+
+/**
+ * Returns the value of some other property on the PageObject.
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continue: alias('continueButton.click')
+ * });
+ *
+ * // calls `page.continueButton.click`
+ * page.continue();
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   continueButton: {
+ *     scope: 'button.continue'
+ *   }
+ *   continueText: alias('continueButton.text')
+ * });
+ *
+ * // checks value of `page.continueButton.text`
+ * assert.equal(page.continueText, 'Go ahead!');
+ *
+ * @example
+ *
+ * const page = PageObject.create({
+ *   clickContinueButton: clickable('button.continue'),
+ *   continue: alias('clickContinueButton')
+ * });
+ *
+ * // calls `clickContinueButton`
+ * page.continue();
+ *
+ * @public
+ *
+ * @param {string} pathToProp - dot-separated path to PageObject property
```

Comment:
Great point. Would you suggest explicitly calling this out as an unsupported use case in the documentation? Even though it's "unsupported," should I add a test for this scenario? That way, if at some point this no longer works, a test failure will sound the alarm and then a decision can be made about whether this use case should continue to be supported or whether the test should simply be removed and the documentation updated accordingly.